packages/react-components/src/components/DetailsCard/DetailsCard.module.scss

```diff
@@ -0,0 +1,83 @@
+$base-class: 'details-card';
+
+.#{$base-class} {
+  position: relative;
+  background-color: var(--surface-primary-default);
+  width: 100%;
+
+  &:hover {
+    .#{$base-class}__float-button {
+      opacity: 1;
+    }
+  }
+
+  &--with-divider {
+    border-top: 1px solid var(--border-basic-tertiary);
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    border: 0;
+    background-color: transparent;
+    padding: 0;
+    padding: var(--spacing-6);
+    width: 100%;
+
+    &:hover {
+      cursor: pointer;
+    }
+
+    &__title {
+      margin: 0;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+    }
+
+    &__icon {
+      transition: all 0.2s ease;
+
+      &--open {
+        transform: rotate(90deg);
+      }
+    }
+
+    &--hide {
+      display: none;
+    }
+  }
+
+  &__float-button {
+    position: absolute;
+    top: var(--spacing-1);
+    right: var(--spacing-1);
+    opacity: 0;
+    z-index: 1;
+  }
+
+  &__content-wrapper {
+    transition: all 0.2s ease;
```

Comment:
I've spotted a strange animation for hidden label section:
https://monosnap.com/file/L6EKItWHu63AEWGKcI8vdFFzTsVCGq
Maybe we can improve it somehow?